libvirt/resource_libvirt_domain_test.go

```diff
@@ -1767,3 +1767,58 @@ func testAccCheckLibvirtDomainDestroy(s *terraform.State) error {
 	}
 	return nil
 }
+func TestAccLibvirtDomain_FwCfgName(t *testing.T) {
+	var domain libvirt.Domain
+	var volume libvirt.StorageVol
+	randomDomainName := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
+	randomFwCfgName := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
+	randomPoolName := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
+	randomPoolPath := "/tmp/terraform-provider-libvirt-pool-" + randomPoolName
+	randomIgnitionName := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
+	var config = fmt.Sprintf(`
+	data "ignition_systemd_unit" "acceptance-test-systemd" {
+		name    = "example.service"
```

Comment:
randomize this name with a number since you copied pasted.

`example.service02` or whatever